### DISCUSSION TITLE
Organisation des sites

### DISCUSSION THREAD

**Discussion by emmanuelbrossier** [dbarzin/mercator#200] [Q&A] [answered]
Bonjour à toutes, tous !

J'ai une question qui bloque quand même pas mal l'alimentation en données de notre base.
Pour faire un rapide résumé, nous sommes une structure découpée entre 4 villes. Le site support est dans l'une de ces villes. On va l'appeler la ville A pour l'exemple.
Seulement, dans cette ville, il y à également plusieurs sous-structures, distantes de quelques kilomètres, mais que l'on doit faire apparaitre dans l'organisation. (Sites - Bâtiments salles)
J'ai commencé à renseigné, au départ, en mettant simplement les villes en tant que site, nous avions donc 4 sites. Puis, dans bâtiments / salles, les différents étages / services / salles de chaque. Mais niveau lisibilité, du fait de la limite de 32 caractère, je me suis retrouvé avec des intitulés de type "NOMBATIMENT-ADMINISTRATION-PER-AFFMED" ayant pour site la ville, avec, pour plus de lisibilité, l'obligation de mettre le détail en description.
J'ai alimenté la moitié des lieux de notre site support, mais l'organisation était "plate" en diagramme.
Avant de finir, j'ai réorganisé mon schéma, et j'ai choisi de remplacer les sites par un nommage de type "2 premières lettres de la ville - BATIMENT - ETAGE" 
C'est déjà plus lisible sur l'organisation, mais j'aurais aimé savoir si il y avait la possibilité de regrouper d'un niveau au dessus, par ville ?
Je ne sais pas si je suis très clair… Dans tous les cas, je pourrais clarifier ma question au besoin ! 

Bonne journée, et merci !

**Comment by dbarzin** [comment, marked as answer]
La structure est sur deux niveaux : site - bâtiments et salles. Vous devez trouver le regoupement qui fait le plus de sens selon votre organisation. Vous pouvez découper les sites par villes ou par villes et sous-structures. 
C'est une bonne occasion de mettre en place une nomenclature pour l'ensemble des salles de l'organisation.
Cela ne devrait pas vous bloquer car vous pouvez par la suite renomer les salles, les équipements qui s'y trouvent y resteront.

**Reply by emmanuelbrossier** [reply]
Tout à fait pour la nomenclature, c'est l'occasion de remettre à jour les listes qui ne le sont pas forcément !
On est parti sur la base de notre GLPI, tout est très détaillé, au moins pour le site support.

La réorganisation semble mieux convenir à l'affichage, nous détaillons tout, même si nous n'auront pas besoin de tout (pour le moment du moins), c'est peut-être une erreur, l'avenir nous le dira ! 
Merci !
